ui/src/keyboardLayouts.ts

```diff
@@ -0,0 +1,42 @@
+import { chars as chars_cs_CZ } from "@/keyboardLayouts/cs_CZ"
+import { chars as chars_en_UK } from "@/keyboardLayouts/en_UK"
+import { chars as chars_en_US } from "@/keyboardLayouts/en_US"
+import { chars as chars_fr_FR } from "@/keyboardLayouts/fr_FR"
+import { chars as chars_de_DE } from "@/keyboardLayouts/de_DE"
+import { chars as chars_it_IT } from "@/keyboardLayouts/it_IT"
+import { chars as chars_nb_NO } from "@/keyboardLayouts/nb_NO"
+import { chars as chars_es_ES } from "@/keyboardLayouts/es_ES"
+import { chars as chars_sv_SE } from "@/keyboardLayouts/sv_SE"
+import { chars as chars_fr_CH } from "@/keyboardLayouts/fr_CH"
+import { chars as chars_de_CH } from "@/keyboardLayouts/de_CH"
+
+type KeyInfo = { key: string | number; shift?: boolean, altRight?: boolean }
+export type KeyCombo = KeyInfo & { deadKey?: boolean, accentKey?: KeyInfo }
+
+export const layouts = {
+  "cs_CZ": "Czech",
```

Comment:
Should these really be in the language they represent? For example `"de_DE": "Deutsch",

Reply:
Yes it could be done and would look like something like this for the above:

```javascript
import { chars as chars_cs_CZ, code as code_cs_CZ, name as name_cs_CZ } from "@/keyboardLayouts/cs_CZ"

export const layouts = {
  code_cs_CZ: name_cs_CZ,
   ..
}
```

It wouldn't save any characters, but it would make it cleaner in terms of moving the definitions together into the keyboard layout files. Is that what you had in mind?